test/Messages/rlexe.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
All proxy specific test cases are in `test\es6` folder. Please move this test case in that folder.